src/Constants/index.ts

```diff
@@ -0,0 +1,32 @@
+import { Session } from '@/types';
+import { DeepKeys } from '@tanstack/react-table';
+
+export * from './Options';
+
+export const DATA_PER_PAGE = 10;
+export const PAGE_SIZE_OPTIONS = [10, 20, 30, 40, 50];
+
+export const KeysToDeleteBeforeDuplicate: DeepKeys<Session>[] = [
+  'end_time',
+  'start_time',
+  'name',
+  'id',
+  'portal_link',
+  'signup_form_id',
+  'owner_id',
+  'session_id',
+  'platform_link',
```

Comment:
note that we cannot retain this for liveclass flow.. 
same goes for platform_id .